### PR TITLE
csvtomd 0.1.1 (new formula)

### PR DESCRIPTION
**Pull request by zmwangx**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

---

266 stars: not _too_ shabby. I found I needed this a couple times recently. Needless to say I'll be responsible for formulae I submitted, so no burden on anyone else (this one's just a ~100 line Python script, so no burden to begin with).

### CODE REVIEW

**Review by MikeMcQuaid — APPROVED** [review]

File: Formula/csvtomd.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+  def install
+    virtualenv_create(libexec, "python3")
+    virtualenv_install_with_resources
+  end
+
+  test do
+    csv = testpath/"test.csv"
+    md = testpath/"test.md"
+    csv.write <<-EOS.undent
```
Comment (ilovezfs) [inline Formula/csvtomd.rb]: you should be able to do `csv = <<-EOS.undent.strip` and avoid the `csv.read.strip` below.

File: Formula/csvtomd.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      column 1,column 2
+      hello,world
+    EOS
+    md.write <<-EOS.undent
```
Comment (ilovezfs) [inline Formula/csvtomd.rb]: you should be able to do md = <<-EOS.undent.strip and avoid the md.read.strip below.

Comment (zmwangx) [inline Formula/csvtomd.rb]: Well, having a text file without LF as the last character is against my religion; it's not technically a text file per POSIX definitions.

Comment (ilovezfs) [inline Formula/csvtomd.rb]: it's nice a file if you do `md =` ... instead of `md.write` and `md.read`.

just a string.

Comment (zmwangx) [inline Formula/csvtomd.rb]: Oh I see, you mean I don't need a file here.

Comment (ilovezfs) [inline Formula/csvtomd.rb]: yup :)

Comment (zmwangx) [inline Formula/csvtomd.rb]: Addressed.

**Pull request closed without merging**